src/model.jl

```diff
@@ -620,6 +620,7 @@ function get_value(parameter::ParameterRef)
     end
 end
 
+#TODO This doesn't work
```

Comment:
What does that mean? In what way? Is this related to transfer elasticities in some way?

Reply:
Unrelated :/  . It just means that one can't re-set the consumer value, it's always calculated as the sum of endowments in the Demand.